scripts/biomedical/humanMetabolomeDatabase/tmcf/hmdb_protein_metabolite.tmcf

```diff
@@ -0,0 +1,8 @@
+Node: E:hmdb_protein_metabolite->E1
+typeOf: dcs:Protein

```

Comment:
does this protein not exist already in the graph? If it's already in the graph then you don't need to include the initialization of the protein node. The only thing needed is the second node adding the property associateProtein to the metabolite and referencing the corresponding dcid column in the metabolites file.

associatedProtein: C:hmdb_protein_metabolite->protein_dcid

Reply:
@khoahoang1891999

Reply:
Fixed